rest/admin_api.go

```diff
@@ -230,7 +231,8 @@ func (h *handler) handleGetDbConfig() error {
 			return base.HTTPErrorf(http.StatusNotFound, "database config not found")
 		}
 
-		h.response.Header().Set("ETag", dbConfig.Version)
+		quoteETag := strconv.Quote(dbConfig.Version)
```

Comment:
`strconv.Quote/Unquote` does more than wrap the given string in double quotes - which may not be quite what we need for Etag headers.